public/script.js

```diff
@@ -1019,7 +1019,7 @@ export function displayOnlineStatus() {
         $('.online_status_text').text($('#API-status-top').attr('no_connection_text'));
     } else {
         $('.online_status_indicator').addClass('success');
-        $('.online_status_text').text(online_status);
+        $('.online_status_text').text(t([online_status]));
```

Comment:
If you want to translate a dynamic value, and not a template string, it should be `translate(online_status)`.
I know your code works, but let's use the correct function

Reply:
That actually makes more sense. As it might contain replaced values inside the string.